.github/actions/spelling/patterns.txt

```diff
@@ -1,122 +1,81 @@
 # See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns
 
-# hit-count: 106 file-count: 28
-# Compiler flags
-(?:^|[\t ,"'`=(])-(?:[DLP](?=[A-Z]{2,})|[WXYlf](?=[A-Z]{2,}|[A-Z][a-z]|[a-z]{2,}))
-
-# http|ftp|file URLs
-#(?:\b(?:https?|ftp|file)://)[-A-Za-z0-9+&@#/%?=~_|!:,.;]+[-A-Za-z0-9+&@#/%=~_|]
-
-# hit-count: 48 file-count: 18
-# GitHub SHAs (markdown)
-(?:\[`?[0-9a-f]+`?\]\(https:/|)/(?:www\.|)github\.com(?:/[^/\s"]+){2,}(?:/[^/\s")]+)(?:[0-9a-f]+(?:[-0-9a-zA-Z/#.]*|)\b|)
-
-# hit-count: 29 file-count: 13
-# version suffix <word>v#
-(?:(?<=[A-Z]{2})V|(?<=[a-z]{2}|[A-Z]{2})v)\d+(?:\b|(?=[a-zA-Z_]))
-
-# hit-count: 8 file-count: 3
-# Wikipedia
-\ben\.wikipedia\.org/wiki/[-\w%.#]+
-
-# hit-count: 5 file-count: 2
-# w3
-\bw3\.org/[-0-9a-zA-Z/#.]+
-
-# hit-count: 3 file-count: 3
-# Contributor
-\[[^\]]+\]\(https://github\.com/[^/\s"]+\)
-
-# hit-count: 3 file-count: 3
-# stackexchange -- https://stackexchange.com/feeds/sites
-\b(?:askubuntu|serverfault|stack(?:exchange|overflow)|superuser).com/(?:questions/\w+/[-\w]+|a/)
+# Automatically suggested patterns
+# hit-count: 1004 file-count: 122
+# Ignore any text between inline back-ticks
+`(.{2,}?)`
 
-# hit-count: 2 file-count: 1
-# mailto urls
-\[([-a-zA-Z=;:/?%&0-9+@.]{3,})]\(mailto:\g{-1}\)
+# hit-count: 338 file-count: 107
+# https/http/file urls
+(?:\b(?:https?|ftp|file)://)[-A-Za-z0-9+&@#/%?=~_|!:,.;]+[-A-Za-z0-9+&@#/%=~_|]
```

Comment:
Hmm, this was commented out previously, I'll revert this -- I had meant to undo this before making the PR...